os/include/tinyara/wifi/rtk/wifi_constants.h

```diff
@@ -633,10 +633,10 @@ enum _WIFI_EVENT_INDICATE{
 	WIFI_EVENT_IP_CHANGED = 16,
 	WIFI_EVENT_ICV_ERROR = 17,
 	WIFI_EVENT_CHALLENGE_FAIL = 18,
-	
-		/* csi rx done event */
-	WIFI_EVENT_CSI_DONE = 160,
-	
+  /* csi rx done event */
+  WIFI_EVENT_CSI_DONE = 19,

```

Comment:
Could you align this as others? We use a tab for indentation. Because this is not in the board (or chip) specific folders, we should follow TizenRT coding rules.

Reply:
Dear @sunghan-chang,
Made the relevant changes.